benchmarking/data/real_data/cosc499_s2023_provider/providers.py

```diff
@@ -0,0 +1,59 @@
+import json
+from os import path
+from typing import List
+
+import numpy as np
+
+from api.models.student import Student, StudentSerializer
+from api.models.team import TeamSerializer
+from api.models.team_set import TeamSet
+from benchmarking.data.interfaces import (
+    StudentProvider,
+    TeamConfigurationProvider,
+)
+
+
+class COSC499S2023InitialTeamConfigurationProvider(TeamConfigurationProvider):
+    def get(self) -> TeamSet:
+        """
+        Returns the teams that were created for the W2021T2 COSC341 class
```

Comment:
nit; sus comment

Reply:
Not sure why this is sus unless you just want it to say TeamSet rather teams

Reply:
the class is wrong on both

Reply:
sorry should've been more specific in review haha